README.md

```diff
@@ -1 +1,13 @@
-# Hello-Microverse-project
\ No newline at end of file
+Hello Microverse
+
+this proyect its the first proyect to practice Create branches with different base branches Understand the concept of version control and Follow GitHub flow.
+
+Built css/html.
+
+Technologies used gut/github/ linter.
+
+👤 Pierre Antonio Barba
+
+GitHub: https://github.com/PierreBarba LinkedIn: PierreBarba
+
+Show your support Give a ⭐️ if you like this project!
```

Comment:
- [ ] Good job adding a `Readme` file 😺, but it doesn't look professional, kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md), and adapt it to the information related to your project 🦖.